cardano-sl.cabal

```diff
@@ -573,6 +578,7 @@ test-suite cardano-test
                        Test.Pos.Block.Logic.VarSpec
 
                        -- Everything else
+                       Test.Pos.CborSpec
```

Comment:
This module includes a spec which tests _all_ the types which were ported on the new serialisation schema I could get my hands on. Probably in the future we should be looking at integrating what me and Andrzej wrote here in the `BinarySpec` scattered across the codebase, although, in practice, I did find extremely useful having all the serialisation-related tests in one place, as it makes interactive testing (via ghci) very quick.